lib/youtube.js

```
@@ -52,63 +55,114 @@ async function searchYT(q) {
+            vQuality: "720",
+            aFormat: "mp3",
+            filenamePattern: "classic",
+            isAudioOnly: true
+        }, {
+            headers: {
+                'Accept': 'application/json',
+                'Content-Type': 'application/json'
+            }
+        });
+
+        if (response.data && response.data.url) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)